File: scripts/tests/tmp_fix.py
import re

with open('panel/src/services/customerService.ts', 'r', encoding='utf-8') as f:
    content = f.read()

# Fix (x) to (x: any)
content = re.sub(r'\(x\)', '(x: any)', content)

# Fix calculateCEI
content = re.sub(r'calculateCEI\(([^,]+),\s*([^,]+),\s*[^)]+\)', r'calculateCEI(\1, \2)', content)

with open('panel/src/services/customerService.ts', 'w', encoding='utf-8') as f:
    f.write(content)

with open('panel/src/services/aiService.ts', 'r', encoding='utf-8') as f:
    content = f.read()

content = content.replace('${res?.skippedDuplicate || skippedDuplicate}', '${skippedDuplicate}')
content = content.replace('${res?.added || added}', '${added}')

with open('panel/src/services/aiService.ts', 'w', encoding='utf-8') as f:
    f.write(content)

with open('panel/src/services/apiSyncService.ts', 'r', encoding='utf-8') as f:
    content = f.read()
content = content.replace('customerState.usingSeedData = false;', '')
content = content.replace('customerState.usingSeedData = true;', '')
with open('panel/src/services/apiSyncService.ts', 'w', encoding='utf-8') as f:
    f.write(content)

print("Done")

File: scripts/tests/tmp_fix.js
const fs = require('fs');

// Fix customerService.ts
let content = fs.readFileSync('panel/src/services/customerService.ts', 'utf-8');
content = content.replace(/\\(x\\)/g, '(x: any)');
content = content.replace(/calculateCEI\\(summary\\.totalCollections \\+ summary\\.totalCreditNotes, summary\\.totalSales, summary\\.netReceivables\\)/g, 'calculateCEI(summary.totalCollections + summary.totalCreditNotes, summary.totalSales)');
fs.writeFileSync('panel/src/services/customerService.ts', content);

// Fix aiService.ts
let aiContent = fs.readFileSync('panel/src/services/aiService.ts', 'utf-8');
aiContent = aiContent.replace('**${res?.skippedDuplicate || skippedDuplicate} Adet**', '**${skippedDuplicate} Adet**');
aiContent = aiContent.replace('**${res?.added || added} Adet**', '**${added} Adet**');
fs.writeFileSync('panel/src/services/aiService.ts', aiContent);

// Fix apiSyncService.ts
let apiContent = fs.readFileSync('panel/src/services/apiSyncService.ts', 'utf-8');
apiContent = apiContent.replace('customerState.usingSeedData', 'false');
fs.writeFileSync('panel/src/services/apiSyncService.ts', apiContent);
